functions/New-DbaCustomError.ps1

```diff
@@ -0,0 +1,150 @@
+function New-DbaCustomError {
+    <#
+    .SYNOPSIS
+        Creates a user defined message in sys.messages. This command does not support Azure SQL Database.
+
+    .DESCRIPTION
+        This command provides a wrapper for the sp_addmessage system procedure that allows for user defined messages to be added to sys.messages.
+        As noted in https://docs.microsoft.com/en-us/sql/relational-databases/system-stored-procedures/sp-addmessage-transact-sql?view=sql-server-ver15#remarks the english message must be present before the non-english messages can be added.
```

Comment:
This URL is pointed at a specific version of SQL Server. Do we need to validate the version for this reference?

Instead of providing long links I think we can get by with justing providing a statement like:

```
Note: See the documentation on sp_addmessage regarding adding non-English messages; the English message must be present.
```